src/index.js

```diff
@@ -1,6 +1,87 @@
-/** *** */ (() => { // webpackBootstrap
-/** *** */
+const apiConfiguration = {
+  baseUrl: 'https://us-central1-js-capstone-backend.cloudfunctions.net/api',
+  gameId: '',
+};
 
-/** *** */
-/** *** */
-/** *** */ })();
+const cleanFields = () => {
+  const user = document.querySelector('input[type=name');
+  const score = document.querySelector('input[type=number');
+  user.value = '';
+  score.value = '';
+};
+
+const createGame = async (name) => {
+  const response = await fetch(`${apiConfiguration.baseUrl}/games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name }),
+  });
+
+  const content = await response.json();
+  const data = content.result.split(' ');
+  apiConfiguration.gameId = data[data.length - 2];
+  return content;
+};
```

Comment:
[Optional]
- [ ] It is bad practice to create a new game for each run of the application, this will pollute the endpoint with lots of game entries. And I will not be able to access my leaderboard after the application closed (There is no way for to end-user to see their game id).  Yous should create one game and use it or give users instructions to get and configure a game id  (API key).